src/dashboard/src/components/archival_storage/tests/test_archival_storage.py

```diff
@@ -43,6 +44,16 @@
 JSON_MIME = "application/json"
 
 
+@pytest.fixture
+def amsetup(db):
+    setting, _ = DashboardSetting.objects.get_or_create(
+        name="dashboard_uuid", defaults={"value": str(uuid.uuid4())}
+    )
+    return {
+        "uuid": setting.value,
+    }
```

Comment:
I'm hoping this fixture could eventually be exported by conftest and contain other installation attributes.